CleverTap/Runtime/Native/UnityNativeWrapper/Models/UnityNativeRaisedEventQueue.cs

```diff
@@ -8,7 +8,8 @@ internal class UnityNativeRaisedEventQueue : UnityNativeBaseEventQueue {
 
         protected override string QueueName => "RAISED_EVENTS";
 
-        internal UnityNativeRaisedEventQueue(UnityNativeCoreState coreState, UnityNativeNetworkEngine networkEngine, int queueLimit = 49, int defaultTimerInterval = 1) : base(coreState, networkEngine, queueLimit, defaultTimerInterval) { }
+        internal UnityNativeRaisedEventQueue(UnityNativeCoreState coreState, UnityNativeNetworkEngine networkEngine, UnityNativeEventValidator eventValidator, int queueLimit = 49, int defaultTimerInterval = 1) : 
```

Comment:
The eventValidator seems unused here. Why do we need it?

Reply:
Removed it.